src/GraphQL/Execution/ExecutionStrategy.cs

```diff
@@ -15,7 +15,7 @@ public abstract class ExecutionStrategy : IExecutionStrategy
     {
         /// <summary>
         /// Executes a GraphQL request and returns the result. The default implementation builds the root node
-        /// and passes execution to <see cref="ExecuteNodeTreeAsync(ExecutionContext, ObjectExecutionNode)"/>.
+        /// and passes execution to <see cref="ExecuteNodeTreeAsync(ExecutionContext, ExecutionNode)"/>.
```

Comment:
Why did you change ObjectExecutionNode to ExecutionNode ?

Reply:
Because the subscription executer is not always executing an object execution node.  Array or value execution nodes are valid and supported.

Reply:
This supports passing a serial or parallel ExecutionStrategy to the SubscriptionExecutionStrategy